manifests/guestcluster/1.17/pvcsi.yaml

```diff
@@ -147,6 +147,7 @@ spec:
             - "--supervisor-fss-namespace=$(CSI_NAMESPACE)"
             - "--fss-name=internal-feature-states.csi.vsphere.vmware.com"
             - "--fss-namespace=$(CSI_NAMESPACE)"
+            - "--use-gocsi=false"
```

Comment:
Are we disabling gocsi by default? I thought we are enabling it by default and will disable it after extensive testing.